sunpy/net/jsoc/attrs.py

```diff
@@ -48,8 +48,8 @@ def __init__(self, start, end=None):
             self.start = start.start
             self.end = start.end
         else:
-            self.start = start if isinstance(start, astropyTime) else astropyTime(parse_time(start))
-            self.end = end if isinstance(end, astropyTime) else astropyTime(parse_time(end))
+            self.start = parse_time(start)
```

Comment:
this whole class can go away. Just import `Time` like `
from sunpy.net.vso.attrs import Time`

Reply:
Do you want

```python

__all__ = [... 'Time', ...]

from sunpy.net.vso.attrs import Time
```
in `jsoc.attrs`?

Reply:
yes please.

Reply:
Might be worth deprecating it, but probably best done separately from here.